### PR TITLE
Supplierrating Documentation

### PR DESCRIPTION
**Pull request by TM-Simplic**

### CODE REVIEW

File: support/SimplicStudio/SupplierRating/fillfieldservice_interface.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    def get_DisplayName(self):
+        return 'Always20'
+        
+    def FillFieldValue(self, field):
+        user_id = Sql.execute('SELECT MAX(Ident) Id from ESS_MS_Intern_User')
+    
+        field.IntegerValue = user_id[0].Id
+        return True    
+
+DependencyInjection.register_instance(IFillFieldValueService, 'Always20', Always20())
```
Comment (simplicbe) [inline support/SimplicStudio/SupplierRating/fillfieldservice_interface.md:32]: Maybe rename `Always20` to `SampleFillService`.

Furthermore, I would add the following sentence: `Services should be registered in the main.py file`.

File: support/SimplicStudio/toc.yml [hunk 2]
```diff
@@ -108,3 +108,8 @@
     href: Troubleshooting/license.md
   - name: Log sql queries
     href: Troubleshooting/Sql-Log.md
+- name: LieferantenBewertung
```
Comment (simplicbe) [inline support/SimplicStudio/toc.yml:111]: `LieferantenBewertung` --> `Supplier rating`?

File: user/Simplic Studio/supplierrating.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+In der Lieferantenbewertung können Sie für Ihre Kontakte, wie zum Beispiel Lieferanten, bewerten.
```
Comment (simplicbe) [inline user/Simplic Studio/supplierrating.md:1]: I would remove the word `für`?

**Review by simplicbe — CHANGES_REQUESTED** [review]

**Review by simplicbe — APPROVED** [review]

File: support/SimplicStudio/SupplierRating/fillfieldservice_interface.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+class SampleFillService(IFillFieldValueService):
+
+    def get_ServiceName(self):
+        return 'SampleFillService'
+
+
+    def get_DisplayName(self):
+        return 'SampleFillService'
+        
+    def FillFieldValue(self, field):
```
Comment (simplicbe) [inline support/SimplicStudio/SupplierRating/fillfieldservice_interface.md:27]: @TM-Simplic we need to adjust the documentation here.

Comment (TM-Simplic) [reply support/SimplicStudio/SupplierRating/fillfieldservice_interface.md:27]: @simplicbe What would you change/add?

**Pull request merged**